### PR TITLE
Protobuf and lvgl

### PR DESCRIPTION
**Pull request by fluffyironboots**
Implemented lvgl in your new protobuf version. Also changed the vendor id in your example application for windows users because of case sensitivity.

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -109,3 +110,12 @@ build_flags =
   ; GPIO >= 34 are input only
   ; (SOC_GPIO_VALID_GPIO_MASK & ~(0ULL | _FL_BIT(34) | _FL_BIT(35) | _FL_BIT(36) | _FL_BIT(37) | _FL_BIT(38) | _FL_BIT(39)))
   -DSOC_GPIO_VALID_OUTPUT_GPIO_MASK=0x30EFFFFFF
+  -DLV_LVGL_H_INCLUDE_SIMPLE=1
+	-DLV_CONF_SKIP=1
+	-DLV_COLOR_DEPTH=16
+	-DLV_TICK_CUSTOM=1
+	-DLV_TICK_CUSTOM_INCLUDE=\"Arduino.h\"
+	-DLV_TICK_CUSTOM_SYS_TIME_EXPR=(millis())
```
Comment (scottbez1) [inline platformio.ini:118]: This causes build errors for me on Linux (I think the parenthesis are messing with shell invocations from pio). Do you mind checking if this alternative works on Windows:
```suggestion
	-DLV_TICK_CUSTOM_SYS_TIME_EXPR=millis\\(\\)
```

**Review by scottbez1 — COMMENTED** [review]
Amazing, thanks for getting this up and running!

My plan will be to create a new dev branch for ongoing LVGL work and then merge this into that branch rather than master for now. That way we can get this merged sooner rather than later so it's easier for others to start tinkering with it, and we don't need to worry about getting it to feature parity with the existing display code first or worry too much about performance yet.

File: platformio.ini (hunk 1 above)
Comment (fluffyironboots) [reply platformio.ini:118]: The double backslash sadly doesnt work under windows. Could you try with a single backslash under linux? Someday we will surely find a combination that works on both systems.

```suggestion
	-DLV_TICK_CUSTOM_SYS_TIME_EXPR=millis\(\)
```

Comment (scottbez1) [reply platformio.ini:118]: Single backslash doesn't work on linux. But I _think_ we don't actually need to specify the LV_TICK_CUSTOM_INCLUDE or LV_TICK_CUSTOM_SYS_TIME_EXPR as long as LV_TICK_CUSTOM is set to 1.

**Pull request merged**